community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -322,6 +348,52 @@ def ignore_err(e) -> bool:
     )
 
 
+def sync_mig_groups():
+    lkp = lookup()
+    zone = lkp.zone.split("/")[-1]
```

Comment:
`lkp.zone` probably refers to the zone of the given VM (one that runs script), so it's controller.
Instead we should fetch for all MIGs in all zones in the region.